### PR TITLE
add a blog: Install Kubernetes 1.22 and containerd the Easy Way

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <feynman@kubesphere.io>

See https://github.com/kubesphere/kubekey/issues/644.

### CODE REVIEW

File: content/en/blogs/install-kubernetes-containerd.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+![k8s-containerd](/images/blogs/en/kubekey-containerd/k8s-containerd.png)
+
+[KubeKey](https://github.com/kubesphere/kubekey) is a lightweight and turn-key installer that supports the installation of Kubernetes, KubeSphere and related add-ons. Writtent in Go, KubeKey enables you to set up a Kubernetes cluster within minutes. In this blog, we will install Kubernetes 1.22 and [containerd](https://containerd.io/) in one command with KubeKey.
+
+## Step 1: Prepare a Linux Machine
+
+To get started with all-in-one installation, you only need to prepare one host according to the following requirements for hardware and operating system.
+
+### Hardware Recommendations
+
```
Comment (FeynmanZhou) [inline content/en/blogs/install-kubernetes-containerd.md:20]: @pixiake what are the minimum hardware requirements If we only install Kubernetes?

Comment (24sama) [reply content/en/blogs/install-kubernetes-containerd.md:20]: Hi @FeynmanZhou 
You can refer to this [doc](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#before-you-begin)

Comment (FeynmanZhou) [reply content/en/blogs/install-kubernetes-containerd.md:20]: Tks, updated.

**Pull request merged**